### ISSUE TITLE
COCO training and evaluation

### ISSUE THREAD

**Issue by andyhahaha** [luuuyi/RefineDet.PyTorch#33]
Do anyone successfully train on coco and reach the paper accuracy? (refinedet 320 VGG: 29.4)
Can somebody provide the coco evaluation code? I want to train coco myself but I can't run evaluation. Thanks!

**Comment by dd604** [comment]
You can use the published code https://github.com/dd604/refinedet.pytorch.git for coco training and evaluation.
